bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Page.java

```diff
@@ -347,4 +347,12 @@ default String getExportedType() {
         throw new UnsupportedOperationException();
     }
 
+    /**
+     * @see ContainerExporter#getMainContentSelector()
```

Comment:
Javadocs should be updated with a proper description. And also: `ContainerExporter#getMainContentSelector()` does not exist and should be removed.